dev/com.ibm.ws.jaxrs.2.1.common/src/com/ibm/ws/jaxrs21/sse/LibertySseEventSinkImpl.java

```diff
@@ -121,6 +122,9 @@ public boolean isClosed() {
                     response.getOutputStream().flush();
 
                     return CompletableFuture.completedFuture(eventContents);
+                } catch (InternalServerErrorException ex) {
+                    handleException(ex, future, event);
+                    throw new IllegalArgumentException("No suitable message body writer for OutboundSseEvent created with data " + event.getData() + " and mediaType " + event.getMediaType() + ". The data contained within the OutboundSseEvent must match the mediaType."); 
```

Comment:
My main concern here is that there are different scenarios that might also cause an `InternalServerErrorException` other than no suitable MBW.  Is it possible to be more specific in checking the exception to ensure that the reason for the exception is due to no suitable MBW?